### PR TITLE
Enhancement/190 change text in delete modal

### PR DESCRIPTION
**Pull request by tanmaythole**

### CODE REVIEW

File: src/Components/DeleteConfirmationModal/index.jsx [hunk 1]
```diff
@@ -4,32 +4,34 @@ import PropTypes from "prop-types";
...
+  handleClose,
+  onDelete,
+}) => (
+  <CustomModal
+    show={show}
+    title={title ? title : "Are You Sure?"}
+    onClose={handleClose}
+  >
+    <CustomModal.Body>
+      {text ? text : "Are you sure you want to DELETE this item?"}
```
Comment (ChinmaySonawane) [inline src/Components/DeleteConfirmationModal/index.jsx]: can we add default value for `text` instead of checking?

Comment (tanmaythole) [reply src/Components/DeleteConfirmationModal/index.jsx]: 👍🏻

File: src/Components/DeleteConfirmationModal/index.jsx [hunk 2]
```diff
@@ -4,32 +4,33 @@ import PropTypes from "prop-types";
...
+  </CustomModal>
+);
 
 DeleteConfirmationModal.propTypes = {
   show: PropTypes.bool,
-  title: PropTypes.string,
-  handleClose: PropTypes.func,
-  onDelete: PropTypes.func,
-  text: PropTypes.string,
+  title: PropTypes.string.isRequired,
```
Comment (ChinmaySonawane) [inline src/Components/DeleteConfirmationModal/index.jsx]: now the time is optional

Comment (tanmaythole) [reply src/Components/DeleteConfirmationModal/index.jsx]: 👍🏻

**Review by ChinmaySonawane — APPROVED** [review]

**Pull request merged**